plugins/a11y-keyboard/Makefile.am

```diff
@@ -55,6 +61,7 @@ liba11y_keyboard_la_CPPFLAGS = \
 liba11y_keyboard_la_CFLAGS =		\
 	$(SETTINGS_PLUGIN_CFLAGS)	\
 	$(LIBNOTIFY_CFLAGS)		\
+	$(LIBATSPI_CFLAGS)		\
```

Comment:
The use of this variable is conditional

Reply:
It makes sense for semantics, but FWIW if HAVE_LIBATSPI is not set it won't be either and thus will expand to an empty value.

Will fix anyway.